generic/tests/cfg/nfs_corrupt.cfg

```diff
@@ -6,7 +6,6 @@
     start_vm = no
     images += " stg"
     image_size_stg = "10G"
-    image_format_stg = "qcow2"
```

Comment:
do we need a 'variants' in cfg to define image_format_stg as qcow2/raw/luks? so no matter what system image format we use, we could test all formats, now the image_format_stg is the same as image1

Reply:
For disk format test loops, I will cover qcow2/raw/luks, so it's OK that the image_format_stg is the same as image1. But it's better to add a 'variants' in cfg to define image_format_stg as qcow2/raw/luks, I will update it. Many thanks.